modules/global.d.ts

```diff
@@ -9,6 +9,7 @@
 
 declare module Liferay {
 	export const FeatureFlags: {[key: string]: boolean};
+	export const SPA: any;
```

Comment:
@bryceosterhaus I'm summoning you because of this line. Is this the best way to prevent TS error?

Reply:
yeah this seems fine, unless we want to add types for the individual items on SPA, but I think a generic any is fine